src/components/Editor/index.js

```diff
@@ -273,16 +259,16 @@ const Editor = React.createClass({
     const bp = breakpointAtLine(this.props.breakpoints, line);
     GutterMenu({ event, line, bp,
       toggleBreakpoint: this.toggleBreakpoint,
-      showConditionalPanel: this.showConditionalPanel,
+      showConditionalPanel: this.toggleConditionalPanel,
       toggleBreakpointDisabledStatus: this.toggleBreakpointDisabledStatus,
       isCbPanelOpen: this.isCbPanelOpen(),
       closeConditionalPanel: this.closeConditionalPanel
     });
   },
 
-  showConditionalPanel(line) {
+  toggleConditionalPanel(line) {
     if (this.isCbPanelOpen()) {
-      return;
+      return this.closeConditionalPanel();
```

Comment:
fixes a bug where we were not closing the panel on subsequent <cmd>+shift+b calls